get_download_info_tester.py

```diff
@@ -0,0 +1,12 @@
+from pymatgen.ext.matproj import MPRester, TaskType
+import os
+material_ids = ["mp-32800", "mp-23494"]
+task_types = [TaskType.GGA_OPT, TaskType.GGA_UNIFORM]
+file_patterns = ["vasprun*", "OUTCAR*"]
+with MPRester(os.environ["MP_API_KEY"]) as mpr:
```

Comment:
`MPRester(os.environ["MP_API_KEY"])` can be left simply as `MPRester()` and it will pick up the API key from the environment.

What is the purpose of this file otherwise? Is it better suited for `dev_scripts`, or a test?